spec/system/discussion_spec.rb

```diff
@@ -0,0 +1,20 @@
+require "rails_helper"
+require_relative "../modules/helpers"
+
+RSpec.describe "Discussion", type: :system do
+  include SessionHelpers
+  include CategoryHelpers
+  include DiscussionHelpers
+
+  it "can be created" do
+    create_discussion("Test Discussion Title", "Test discussion body")
+  end
+
+  it "can be edited" do
+    edit_discussion("Test Discussion Title", "Test discussion body")
+  end
+
+  it "can be deleted" do
+    delete_category("Test Discussion Title", "Test discussion body")
+  end
```

Comment:
This, coupled with my comments about retesting code paths the long way around, makes me think the helpers are too far extracted.  What's being tested here is not clear anymore.  What if the helpers are only facilitating the steps, but the expectations are moved to the correct spec?  You can still take advantage of custom matchers here, but what's being tested will be much more clear.